www/install/steps/process/createDbUser.php

```diff
@@ -90,11 +94,29 @@
     }
     // creating the user
     $prepareCreate->execute();
+
+    // checking mysql version before trying to alter the password plugin
+    $prepareCheckVersion = $link->prepare($checkMysqlVersion);
+    $prepareCheckVersion->execute();
+    while ($row = $prepareCheckVersion->fetch()) {
+        if (!isset($versionNumber) && $row['Variable_name'] === "version") {
+            $versionNumber = $row['version'];
+        } elseif (!isset($versionName) && $row['Variable_name'] === "version_comment") {
+            $versionName = $row['version_comment'];
+        }
+    }
+    if ((strpos($versionName, "MariaDB") !== false && version_compare($versionNumber, '10.2.0') >= 0)
```

Comment:
I'm not sure that we need to ALTER the centreon's user, as the deprecated "plugin" problem should affect only mysql >= 8.0